### DISCUSSION TITLE
New SBC MedoaTek Genio 1200 Radxa NIO 12L

### DISCUSSION THREAD

**Discussion by NicholasFlamy** [blakeblackshear/frigate#10989] [General]
Okay, so the chip is about 2 years old but the SBC is brand new and the chip is supposedly better than the Rockchip RK3588 which is in all of the Raspberry Pi 5-beating SBCs. It has 4 Cortex-A78 and 4 Cortex-A55 cores, up to 16GB of LPDDR4X, and up to 512GB UFS storage. It also has a 4 TOPs NPU, 4Kp90 AV1/VP9/HEVC/H.264 decoder, and 4Kp60 HEVC/H.264 encoder.
Link:
https://radxa.com/products/nio/12l/

So when I saw this and searched the chip and saw it supposedly beats the RK3588 I immediately though, this would make an amazing Battery Powered Frigate Camera (with a big solar panel as well.)

Oh also, did I mention, there is Ubunut certification so hopefully that means good Linux support. So I'm thinking about purchasing this and possibly helping to support AI acceleration in Frigate with it because of the Community Supported Boards.

**Comment by NicholasFlamy** [comment]
RK3588 NPU is 6 TOPs so it's faster for AI but overall they seem to be pretty similar I guess.

**Comment by nyanmisaka** [comment]
The lack of FFmpeg support would be a huge disadvantage for this MTK chip, which is what is needed by frigate. Their BSP uses Gstreamer.

**Reply by NicholasFlamy** [reply]
Oh, didn't know about this. Thank you for pointing this out. I liked this SoC because of the Cortex-A78 cores but I think I'm going to get something with an RK3588.
